tox.ini

```diff
@@ -7,11 +7,7 @@ setenv =
     PYTHONPATH = {toxinidir}
 
 deps =
-    django111: Django>=1.11,<2.0
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
-    django22: Django>=2.2,<2.3
```

Comment:
Do not remove this one and add the django30 option. You should remove from test.txt the line with the django version in order to control which version is used, See https://github.com/edx/edx-enterprise/blob/6f9b8535cd68ee092eb389d53f69b6e05aa2e3a2/Makefile#L92 for reference